packages/lodestar/src/chain/attestation/processor.ts

```diff
@@ -64,7 +64,7 @@ export async function processAttestation(
         balances[i]
       );
     }
-    logger.verbose(`Attestation ${toHexString(attestationHash)} passed to fork choice`);
+    // logger.verbose(`Attestation ${toHexString(attestationHash)} passed to fork choice`);
```

Comment:
this log is quite annoying bc it appears too much, not sure if we want to keep it

Reply:
Maybe put it to debug? Generally it's somewhat usefull to see if attestations are managing to get into fork choice